internal/controller/feature/features.go

```diff
@@ -27,56 +27,78 @@ import (
 	utilfeature "github.com/cert-manager/cert-manager/pkg/util/feature"
 )
 
+// see https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages
+
 const (
-	// alpha: v0.7.2
+	///////////////////////////////////////////////////////////////////////
+	// Owner: @username
+	// Alpha: vX.Y
+	// Beta: ...
+	//
+	// FeatureName will enable XYZ feature.
+	// Fill this section out with additional details about the feature.
+	// FeatureName featuregate.Feature = "FeatureName"
+	///////////////////////////////////////////////////////////////////////
```

Comment:
nitpick: I like having a template here - but this was confusing to me at first because it looks like a left over comment or like a placeholder waiting to be filled in. Could we add a comment explaining that this is a template to be copied and pasted?

Reply:
fixed in https://github.com/cert-manager/cert-manager/pull/6299/commits/1795c1985f2c9b96a6fbf3f7b0ed3966b767b4e3